data/core/init.lua

```diff
@@ -953,9 +953,11 @@ function core.show_title_bar(show)
 end
 
 
-function core.add_thread(f, weak_ref)
+function core.add_thread(f, weak_ref, ...)
   local key = weak_ref or #core.threads + 1
-  local fn = function() return core.try(f) end
+  local args = {...}
+  local table_unpack = rawget(_G, 'unpack') or rawget(_G.table, 'unpack')
```

Comment:
There's no need for this, just use `table.unpack`.

Reply:
Ok, I remove this, Lua 5.4 is the only version of build then this is ok

Reply:
We do have this https://github.com/lite-xl/lite-xl/blob/0a70b13a734031b1e36c8fd7b0d55cf104da0f60/data/core/start.lua#L33-L34 that ensures that `table.pack`/`table.unpack` always work.